web/tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "strict" : true,
+    "forceConsistentCasingInFileNames": true,
```

Comment:
Why are these compiler options needed in the context of this change? If they are really needed, I prefer a specific PR for them documenting the reason in the PR description.

Reply:
VSCode was showing some errors while I was working on the code and it recommended me to use those two options.
Stackoverflow also recommended those settings for writing a overall better code.
I am sorry regarding the PR. I will be sure to keep it in mind for the future.
Thank you.

Reply:
Ok, I've checked the [TypeScript Config documentation](https://www.typescriptlang.org/tsconfig) and the [`forceCosistentCasingInFileNames`](https://www.typescriptlang.org/tsconfig#forceConsistentCasingInFileNames) definitely makes sense for avoiding import errors when working with case-insensitive filesystems as it is the case of Windows (read more at https://learn.microsoft.com/en-us/windows/wsl/case-sensitivity#differences-between-windows-and-linux-case-sensitivity)

I've doubts about using [`strict`](https://www.typescriptlang.org/tsconfig#strict), I need to read it with more attention and do some tests before going ahead. 

Anyway, let's keep this file out of this PR.

Thanks a lot for letting us know about these VSCode suggestions you found :100: